### ISSUE TITLE
BLE Library and HTTP Library Interference

### ISSUE THREAD

**Issue by HHavendano** [nkolban/esp32-snippets#1136]
Hi, i've a proyect with both library. i've a code that connects to bluetooth and connects to Wi-Fi. Both work ,but not the way that i want. The codes for each library are in "setup". First i put the Bluetooth's code then the HTTPS, the problem of this is that the Bluetooth works ,but the HTTPS doesn't (it can't download a file from a website). And if i put the HTTPS over the BLE's code. HTTPS works, but BLE doesn't (I try to send a massage to ESP32 ,but it doesn't receive). 

What could be the problem? ,and what can i do?

**Comment by chegewara** [comment]
memory shortage?

**Comment by HHavendano** [comment]
I don't think so, i was investigating in these hours. And i found a comment that "the Bluetooth and Wifi use the same antenna or something like that" ,and that it's difficult to use it at the same time. @chegewara Do you have any idea?

**Comment by chegewara** [comment]
Yes, my idea is memory shortage, because i never had any issue to use wifi and ble at the same time, and i did many such projects, including the one i am working with now. I just had to switch to NimBLE, because of memory shortage this time, and only this time, but this is huge project, including LTE.

**Comment by HHavendano** [comment]
NimBLE? , My proyect is huge too.

**Comment by chegewara** [comment]
https://github.com/h2zero/NimBLE-Arduino

**Comment by HHavendano** [comment]
@chegewara Thanks, I'll try change the libraries.

**Comment by HHavendano** [comment]
@chegewara I have problems to adapt it. i use 
#include <BLEDevice.h>
#include <BLEUtils.h>
#include <BLEServer.h>
#include <BLE2902.h>
